README.md

```diff
@@ -86,6 +86,7 @@
   - [Passport Email Auth](https://github.com/marcomelilli/nestjs-email-authentication) - Starter project that includes API for user email authentication with MongoDB and PassportJs.
   - [NestJS Project Structure](https://github.com/CatsMiaow/node-nestjs-structure) - Example of constructing a project structure with NestJS.
 - Boilerplate
+  - [NestJS Permission Boilerplate](https://github.com/Ferdysd96/nestjs-permission-boilerplate) - This is a basic NestJS boilerplate project built on the more powerful nodeJS framework. The main purpose of this project is to dynamically handle roles and permissions assigned to the user.
```

Comment:
Hi @Ferdysd96, check the rules:

**Rules:**

NestJS / Nest: It's not nest, nestjs, nest.js, and other variants.
**Node.js: It's not nodejs, node, and other variants.**

_Section:_  ... on the more powerful ~nodeJS~ (should be "Node.js") framework. The main purpose ...

Reply:
Hi, @juliandavidmr can you check it now please